tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -563,6 +563,100 @@ test.describe( 'Nested Tabs tests', () => {
 
 		await cleanup( wpAdmin );
 	} );
+
+	test( 'Verify the correct relationships between normal, hover and active styling', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await setup( wpAdmin );
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Hex colors.
+		const colorGreen = '#95E46E',
+			colorYellow = '#CFE46E',
+			colorBlue = '#134FF2',
+			colorBrown = '#967008',
+			colorRed = '#961708',
+			colorPink = '#E1086E',
+			colorGreenRgb = 'rgb(149, 228, 110)',
+			colorYellowRgb = 'rgb(207, 228, 110)',
+			colorBlueRgb = 'rgb(19, 79, 242)',
+			colorBrownRgb = 'rgb(150, 112, 8)',
+			colorRedRgb = 'rgb(150, 23, 8)',
+			colorPinkRgb = 'rgb(225, 8, 110)';
+
+		// Add widgets.
+		await editor.addWidget( 'nested-tabs', container );
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tab-title.e-normal.e-active' );
+		// Add icons.
+		await setIconsToTabs( page, tabIcons );
+
+		// Normal tab styling: text color green, border color: green and icon color: yellow.
+		await editor.activatePanelTab( 'style' );
+		// Set text color.
+		await setTabItemColor( page, editor, 'section_title_style', 'title_normal', 'title_text_color', colorGreen );
+		// Set border color.
+		await setTabBorderColor( page, editor, 'normal', '', colorGreen, '5' );
+		// Set icon color.
+		await editor.activatePanelTab( 'content' );
+		await setTabItemColor( page, editor, 'icon_section_style', 'icon_section_normal', 'icon_color', colorYellow );
+		await editor.activatePanelTab( 'content' );
```

Comment:
I think that this line is redundant.

Reply:
@davseve I added these lines, to solve a glitch. Somehow the test struggled to switch between icon/title/tab panels. I added these lines to 'reset' to the original position.